helpers/aws/regions_gov.js

```diff
@@ -80,6 +80,8 @@ module.exports = {
     codebuild: regions,
     mq: regions,
     glacier: regions,
+    kendra:  ['us-east-1', 'us-east-2', 'us-west-2','ap-southeast-1',
```

Comment:
@AkhtarAmir is this okay or it should be just regions, can you please verify this.